mne/preprocessing/ica.py

```diff
@@ -120,7 +120,23 @@ class ICA(ContainsMixin):
     u"""M/EEG signal decomposition using Independent Component Analysis (ICA).
 
     This object can be used to estimate ICA components and then remove some
-    from Raw or Epochs for data exploration or artifact correction.
+    from Raw, Epochs or Evoked for data exploration or artifact correction.
```

Comment:
I would never use ICA on `Evoked`, I don't think it should be here. There wouldn't be enough data to train the model on.

Reply:
Oh, you probably mean to train on the Raw/Epochs and then apply on Evoked? Then it should be clearer.

Reply:
Is this OK now or should we rephrase it?

Reply:
OK for me